src/app/store/reducers/app.reducer.ts

```diff
@@ -57,6 +57,10 @@ export function reducer(
         pageTitle
       };
     }
+    // TO add appropriate state as/ when ,reqd.
```

Comment:
Is this to be done or implemented now? sorry its not clear

Reply:
Hmm, actually copied this from xui-webapp. Figured if it's a necessity for webapp, it would be a necessity for manage.
https://github.com/hmcts/rpx-xui-webapp/blob/13b91f6e26aba169ae526b59924ef0d521e7a9ce/src/app/store/reducers/app-config.reducer.ts#L27